aquadoggo/src/db/models/log.rs

```diff
@@ -110,44 +114,83 @@ impl Log {
                 logs
             WHERE
                 author = $1
-                AND schema = $2
+                AND document = $2
             ",
         )
         .bind(author)
-        .bind(schema)
+        .bind(document)
         .fetch_optional(pool)
         .await?;
 
         Ok(result)
     }
 
-    /// Returns registered or possible log_id of an author's schema.
+    /// Returns registered or possible log id for a document.
     ///
-    /// If no log has been found for a USER schema it automatically returns the next unused log_id.
+    /// If no log has been previously registered for this document automatically returns the next
+    /// unused log_id.
     /// SYSTEM schema log ids are pre-defined by the protocol specification.
-    pub async fn find_schema_log_id(pool: &Pool, author: &Author, schema: &Hash) -> Result<LogId> {
-        // Determine log_id for author's schema
-        let schema_log_id = Log::get(pool, author, schema).await?;
+    pub async fn find_document_log_id(
+        pool: &Pool,
+        author: &Author,
+        document: &Hash,
+    ) -> Result<LogId> {
+        // Determine log_id for this document
+        let document_log_id = Log::get(pool, author, document).await?;
 
         // Use result or find next possible log_id automatically
-        let log_id = match schema_log_id {
+        let log_id = match document_log_id {
             Some(value) => value,
             None => Log::next_user_schema_log_id(pool, author).await?,
```

Comment:
Oh yes, that may have been misleading. This PR is not removing the distinction of user and system logs. It would be sensible to also do that before the next release though, to not require another purging of invalidated entries.

The next available log is just the next available _user log_. So if you have used logs 1 and 3 you can't just start publishing entries in log 15, aquadoggo will require that you use up all other user logs first.

This PR is also not establishing new language everywhere so starting here we have a language mix. Could be another thing to tackle soon :)